### PR TITLE
checked out changed nginx files

### PR DESCRIPTION
**Pull request by EdwardAngert**
#1402 and #1450 hopefully actually fixed this time

### CODE REVIEW

File: docs/tools-reference/tools/manipulate-lists-with-sort-and-uniq.md [hunk 1]
```diff
@@ -6,20 +6,20 @@ description: Use the sort and uniq Linux utilities to manage and order
 keywords: ["linux", "common commands", "sort", "uniq", "shell", "bash"]
 license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
 aliases: ['linux-tools/common-commands/sort-uniq/']
-modified: 2018-01-16
+modified: 2018-01-23
```
Comment (GuessWhoSamFoo) [inline docs/tools-reference/tools/manipulate-lists-with-sort-and-uniq.md]: why is this here?

**Comment by cwlinode** [comment]
Closed for https://github.com/linode/docs/pull/1509.
